docs/tracing.md

```diff
@@ -97,6 +97,8 @@ config:
     key_file: ""
     server_name: ""
     insecure_skip_verify: false
+  sampler_type: ""
+  sampler_param: ""
```

Comment:
Even though we take this over from the OTEL specification, maybe we could add a few words about which values are accepted etc. in the text above?